src/writer.rs

```diff
@@ -198,11 +198,11 @@ fn node_name_valid(name: &str) -> bool {
 }
 
 fn node_name_valid_char(c: char) -> bool {
-    matches!(c, '0'..='9' | 'a'..='z' | 'A'..='Z' | ',' | '.' | '_' | '+' | '-')
+    c.is_ascii_alphanumeric() || matches!(c, ',' | '.' | '_' | '+' | '-')
```

Comment:
I'm not sure I understand the commit message. So clippy was failing only for `node_name_valid_first_char`, and not for `node_name_valid_char`?

Reply:
@lauralt Yes, that's correct. I used my initiative to pre-empt what I expect clippy will start complaining about in the future. :)